src/components/flash/flash.js

```diff
@@ -223,19 +233,23 @@ class Flash extends React.Component {
    * @param {Object} ev
    * @return(Void)
    */
-  toggleDialog = (key, ev) => {
-    if (ev) { ev.preventDefault(); }
-
-    let state = this.state.dialogs[key];
-    // open/close the dialog
-    this.setState({ dialogs: { [key]: !state }});
-
-    // start/stop the timer if the dialog opens or closes
-    if (state) {
-      this.startTimeout();
-    } else {
-      this.stopTimeout();
-    }
+  toggleDialog = (key) => {
```

Comment:
One benefit of the fat arrow version is that it makes it very clear that the function takes an argument. Whereas if when using bind, you wouldn't know unless you look at where the function is actually called.